lib/pf/Switch/Ruckus.pm

```diff
@@ -166,6 +174,20 @@ sub deauthTechniques {
 }
 
 
+
+=item returnRoleAttribute
+
+What RADIUS Attribute (usually VSA) should the role returned into.
+
+=cut
+
+sub returnRoleAttribute {
+    my ($self) = @_;
+
+    return 'Ruckus-User-Groups';
```

Comment:
The earliest I could find it in the guides was Release 9.8  for zone director (from early 2014). But the attribute itself is supported in Radius since release 9.1 which is the first release for any of these controllers. The attribute was just used for admin-user's group identification and not for wifi users, so I doubt it's presence would break anything.  On releases prior to 9.8 groups tied to users seem to be limited to only allowing attachment to specific SSIDs, while on 9.8 onwards they allow specifying rate-limits, vlan-overrides, device identification, etc.